PROTO_tests/tests/dataframe_test.py

```diff
@@ -0,0 +1,704 @@
+import os
+import tempfile
+
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal, assert_series_equal
+
+import arkouda as ak
+from arkouda import io_util
+
+
+class TestDataFrame:
+    df_test_base_tmp = "{}/df_test".format(os.getcwd())
+    io_util.get_directory(df_test_base_tmp)
+
+    @staticmethod
+    def build_ak_df():
+        username = ak.array(["Alice", "Bob", "Alice", "Carol", "Bob", "Alice"])
+        userid = ak.array([111, 222, 111, 333, 222, 111])
+        item = ak.array([0, 0, 1, 1, 2, 0])
+        day = ak.array([5, 5, 6, 5, 6, 6])
+        amount = ak.array([0.5, 0.6, 1.1, 1.2, 4.3, 0.6])
+        bi = ak.arange(2**200, 2**200 + 6)
+        return ak.DataFrame(
+            {
+                "userName": username,
+                "userID": userid,
+                "item": item,
+                "day": day,
+                "amount": amount,
+                "bi": bi,
+            }
+        )
+
+    @staticmethod
+    def build_pd_df():
+        username = ["Alice", "Bob", "Alice", "Carol", "Bob", "Alice"]
+        userid = [111, 222, 111, 333, 222, 111]
+        item = [0, 0, 1, 1, 2, 0]
+        day = [5, 5, 6, 5, 6, 6]
+        amount = [0.5, 0.6, 1.1, 1.2, 4.3, 0.6]
+        bi = [2**200, 2**200 + 1, 2**200 + 2, 2**200 + 3, 2**200 + 4, 2**200 + 5]
+        return pd.DataFrame(
+            {
+                "userName": username,
+                "userID": userid,
+                "item": item,
+                "day": day,
+                "amount": amount,
+                "bi": bi,
+            }
+        )
+
+    @staticmethod
+    def build_ak_df_duplicates():
+        username = ak.array(["Alice", "Bob", "Alice", "Carol", "Bob", "Alice"])
+        userid = ak.array([111, 222, 111, 333, 222, 111])
+        item = ak.array([0, 1, 0, 2, 1, 0])
+        day = ak.array([5, 5, 5, 5, 5, 5])
+        return ak.DataFrame({"userName": username, "userID": userid, "item": item, "day": day})
+
+    @staticmethod
+    def build_pd_df_duplicates():
+        username = ["Alice", "Bob", "Alice", "Carol", "Bob", "Alice"]
+        userid = [111, 222, 111, 333, 222, 111]
+        item = [0, 1, 0, 2, 1, 0]
+        day = [5, 5, 5, 5, 5, 5]
+        return pd.DataFrame({"userName": username, "userID": userid, "item": item, "day": day})
```

Comment:
@pierce314159 I was doing this to ensure both methods of dataframe creation are tested, but we could change the test of creation from pandas to test creation from a dictionary and make this update. Does that seem reasonable?

Reply:
Updated these were applicable. I also removed `test_from_pandas` since we test this in `test_dataframe_creation` and many tests will use it via `build_ak_df`.

Reply:
Yah that makes sense to me!